### ISSUE TITLE
resume training option for custom language model

### ISSUE THREAD

**Issue by pushpalatha1405** [flairNLP/flair#2747]
**Is your feature/enhancement request related to a problem? Please describe.**
I did find a way to resume the training from stored checkpoint for the language model.
**Describe the solution you'd like**
In the language_model_trainer.py, there exist a static method for load_checkpoint which return the languageModelTrainer(loss,op_state_dict ,model etc). but this method is of no use further.
also i added to the language_model_trainer.py another static method load_from_checkpoint() as described in the link https://github.com/flairNLP/flair/commit/c25c89f0a94c8c0879d052530502e60f3e8e421a
but  still i cannot use it as it pops up error saying module ot registered and if i pass the checkpoint to the langauageModelTrainer as its dict so i get error its not subscriptable.

**Additional context**
Similar to NER model training resume logic in trainer.py ,is if possible to add  code to the langauge_model_trainer.py.

Please let me know possibility and solutions so that i can add these code patches to enhance the resume training option in language model similar to NER model

regards,
Pushpalatha M
